fish_prompt.fish

```diff
@@ -13,7 +13,7 @@ function fish_prompt
 	__sf_util_set_default SPACEFISH_PROMPT_SUFFIXES_SHOW true
 	__sf_util_set_default SPACEFISH_PROMPT_DEFAULT_PREFIX "via "
 	__sf_util_set_default SPACEFISH_PROMPT_DEFAULT_SUFFIX " "
-	__sf_util_set_default SPACEFISH_PROMPT_ORDER time user dir host git package node ruby golang php rust haskell aws pyenv kubecontext exec_time line_sep battery jobs exit_code char
+	__sf_util_set_default SPACEFISH_PROMPT_ORDER time user dir host git package node ruby golang php rust haskell docker pyenv kubecontext exec_time line_sep battery jobs exit_code char
```

Comment:
Whoops! Looks like you might've accidentally removed `aws` from the prompt order while merging. 😅 

```suggestion
	__sf_util_set_default SPACEFISH_PROMPT_ORDER time user dir host git package node ruby golang php rust haskell docker aws pyenv kubecontext exec_time line_sep battery jobs exit_code char
```

Reply:
I think it got lost while I was resolving conflict.
So this was the reason for the travis build failing. 🧐